client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -179,7 +182,7 @@ class RegistrantExtraInfoCaForm extends React.PureComponent {
 export default connect(
 	state => ( {
 		contactDetailsExtra: getContactDetailsExtraCache( state ),
-		userWpcomLang: getCurrentUserLocale( state )
+		userWpcomLangIsFr: getCurrentUserLocale( state ).match( /^fr-?/i )
```

Comment:
What happens if getCurrentUserLocale returns null? Not introduced here, this is also relevant to current version.